src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx

```diff
@@ -0,0 +1,193 @@
+                            can be provided to users via&nbsp;
+                            <Link to="/global-config/auth/users" className="fs-13 cb-5 lh-20">
+                                User Permissions.
+                            </Link>
+                        </div>
+                    )}
+                </div>
+            </div>
+        )
+    }
+
+    const renderModalBody = () => {
+        const totalApproverText = `${node?.approvalUsers?.length ?? 0} Approvers`
+        return (
+            <div className="trigger-modal__body h-100vh">
+                <div className="material-list__title pb-16">
+                    {selectedTabIndex === 0 ? (
+                        'Request approval for an image to deploy or rollback'
+                    ) : (
+                        <>
+                            {`At least ${
+                                node?.userApprovalConfig?.requiredCount ?? 1
+                            } approvals are required for an image to be deployed. `}
+                            <TippyCustomized
+                                theme={TippyTheme.white}
+                                className="w-300 h-100"
+                                placement="bottom"
+                                Icon={ApproversIcon}
+                                iconClass="fcv-5"
+                                heading={totalApproverText}
+                                additionalContent={getApprovalUsersTippyContent()}
+                                showCloseButton={true}
+                                trigger="click"
+                                interactive={true}
+                            >
+                                <span className="fs-13 dc__underline cursor">{totalApproverText}</span>
+                            </TippyCustomized>
+                        </>
+                    )}
+                </div>
+                <ApprovalMaterial
+                    material={selectedTabIndex === 1 ? approvalRequestedMaterial : remainingMaterial}
+                    envName={node?.environmentName}
+                    stageType={stageType}
+                    changeTab={changeTab}
+                    toggleSourceInfo={toggleSourceInfo}
+                    appId={appId}
+                    pipelineId={pipelineId}
+                    parentEnvironmentName={node?.parentEnvironmentName}
+                    node={node}
+                    selectedTabIndex={selectedTabIndex}
+                />
+            </div>
+        )
+    }
+
+    const renderEmpty = () => {
+        if (selectedTabIndex === 0) {
+            return (
+                <EmptyView
+                    title="No image available"
+                    subTitle="Trigger build pipeline and find the image here"
+                    imgSrc={noartifact}
+                />
+            )
+        }
+
+        return (
+            <EmptyView
```

Comment:
same here

Reply:
As discussed, this will be taken care during UX/Bug fixes.